anyblok_furetui/furetui/resource.py

```diff
@@ -693,12 +705,13 @@ def button_for(self, userid, Model, button, pks):
 
             if resource is None:
                 raise Exception(
-                    f"On resource {self.id}, the resource is not "
-                    f"mapped for {attributes}")
+                    f"On resource {self.identity} : The button "
+                    f"{attributes} defined an unmapped resource "
+                    f"{button.attrib['open-resource']}")
         else:
             raise Exception(
-                f"On resource {self.id}, no action defined on button "
-                f"{attributes}")
+                f"On resource {self.identity} : The button "
```

Comment:
where the self.identity come from ? do you mean ``self.id``

Reply:
nope missing code